Content.Shared/Light/EntitySystems/LightReplacerSystem.cs

```diff
@@ -171,7 +159,7 @@ public bool TryReplaceBulb(EntityUid replacerUid, EntityUid fixtureUid, EntityUi
     ///     Try to insert a new bulb inside light replacer
     /// </summary>
     /// <returns>True if successfully inserted light, false otherwise</returns>
-    public bool TryInsertBulb(EntityUid replacerUid, EntityUid bulbUid, EntityUid? userUid = null, bool showTooltip = false,
+    private bool TryInsertBulb(EntityUid replacerUid, EntityUid bulbUid, EntityUid? userUid = null, bool showTooltip = false,
```

Comment:
For these, I think leaving them as public is fine. It's more nuanced but there may be something in the future that might use this as part of a call to the system.